Oqtane.Shared/Shared/Constants.cs

```diff
@@ -71,6 +71,6 @@ public class Constants {
 
         public static readonly string SatelliteAssemblyExtension = ".resources.dll";
 
-        public static readonly string DefaultCulture = CultureInfo.InstalledUICulture.Name;
+        public static readonly string DefaultCulture = "en";
```

Comment:
It's better to change the default culture into English instead of the current installed one